crates/sui-core/src/transaction_outputs.rs

```diff
@@ -74,6 +76,28 @@ impl TransactionOutputs {
                 }
             });
 
+            // Create markers for each config update. Note that we always place it under the same
+            // object key. This way we can easily query to see if the config marker has already
+            // been updated this epoch.
+            let mutated_config_objects = input_objects
+                .iter()
+                .filter_map(|(id, obj)| {
+                    if obj.struct_tag().is_some_and(|tag| is_config(&tag))
+                        || *id == SUI_DENY_LIST_OBJECT_ID
+                    {
+                        if let Some(((seqno, _), _)) = mutable_inputs.get(id) {
```

Comment:
There was some RFC a while back in Rust for `let-and`, where you could put `if-let` in conditionals... really wishing we had that here